File: conciertosBack/shared/src/main/java/com/rubenazo/buscaConciertos/domain/Concert.java
package com.rubenazo.buscaConciertos.domain;

import java.time.Instant;
import java.time.LocalDate;
import java.util.List;

public record Concert(
    String id,
    String salaConciertoId,
    List<String> artistIds,
    LocalDate date,
    String time,
    String price,
    String sourceUrl,
    Instant updatedAt
) {}

File: conciertosFront/shared/src/commonMain/kotlin/org/rubenazo/conciertosfront/core/domain/model/Concert.kt
package org.rubenazo.conciertosfront.core.domain.model

data class Concert(
    val id: String,
    val salaConcierto: SalaConcierto,
    val artists: List<Artist>,
    val date: String,
    val time: String?,
    val price: String?,
    val sourceUrl: String?,
    val updatedAt: String
)
